sycl/test-e2e/Basic/free_function_queries/free_function_queries.cpp

```diff
@@ -1,5 +1,5 @@
 // FIXME: Investigate OS-agnostic failures
-// REQUIRES: TEMPORARY_DISABLED
+// UNSUPPORTED: overall
```

Comment:
maybe a nitpick but `overall` is a bit unclear to me, maybe like `will_never_work` or `never_expected_to_work`, or maybe something shorter i can't think of

Reply:
I thought about something this, but what if it will work someday? :)

Reply:
i guess we need the code owners to decide that, maybe for now we can do like `UNSUPPORTED: temporary` and ask code reviewers to update it to `permanent` or `never_will_work` or whatever?

Reply:
my assumption was that it's supported on some configurations but not others (will always fail on cuda, should work elsewhere). if the test is unsupported on all platforms and the code owner doesn't expect the change, i agree we should remove the test.